containenv/containenv/cli/commands/init.py

```diff
@@ -21,9 +21,25 @@
 Options:
   -h, --help        Display this help message and exit.
 '''
+import collections
+import docker
 import os
+import logging
+from pkg_resources import resource_filename
```

Comment:
My docker experiments lead me to believe that `pkg_resources` _IS_ part of the standard library as of Python 2..12.